server/controllers/stock/reports/common.js

```diff
@@ -85,7 +85,7 @@ async function getDepotMovement(documentUuid, enterprise, isExit) {
 
   const sql = `
     SELECT
-      i.code, i.text, iu.text AS unit, BUID(m.document_uuid) AS document_uuid,
+      BUID(l.uuid) AS uuid, i.code, i.text, iu.text AS unit, BUID(m.document_uuid) AS document_uuid,
```

Comment:
Can you label this `lot_uuid`?  Otherwise, it gets confused with the `stock_movement`.`uuid`, which what I would expect this function to return.

Reply:
RESOLVED 👍🏽